src/app/components/progress-indicator/index.html

```diff
@@ -0,0 +1,248 @@
+<sky-demo-page
+  pageTitle="Progress indicator"
+  skyuxModule="SkyProgressIndicatorModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The progress indicator component visually represents progress through a series of sequential steps toward a final goal. Active progress indicators represent tasks where user actions are broken into discrete steps, and passive progress indicators represent tasks that concern users but where they do not directly act upon the steps.
+    </p>
+    <p>
+      Active progress indicators can be modal wizards or waterfall pages. For <a routerLink="/components/wizard">modal wizards</a>, place progress indicator components within modal components to guide users through tasks that require a series of related decisions. For waterfall pages, place progress indicator components on pages where records or system settings are in an incomplete state until users complete steps to activate them.
+    </p>
+ </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Progress indicator properties">
+    <sky-demo-page-property
+      propertyName="displayMode()"
+      isOptional="true"
+      defaultValue="Vertical"
+    >
+      Specifies whether to display the progress indicator vertically or horizontally. For <a routerLink="/components/wizard">modal wizards</a>, use the horizontal display mode. For waterfall pages, use the vertical display mode. And for passive progress indicators, we recommend the vertical display mode.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="messageStream"
+      isOptional="true"
+    >
+      Specifies an observable of <stache-code>SkyProgressIndicatorMessageType</stache-code> that determines the status to display for progress items in the progress indicator. The message stream is a queue of commanding messages to change the state of the progress indicator based on one of the three message types.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="startingIndex"
+      isOptional="true"
+    >
+      Specifies the index for the first progress item to make active when the progress indicator loads.
```

Comment:
Might be worth mentioning that all the steps preceding the step of the startingIndex are going to be marked as "complete"